components/swatch/index.css

```diff
@@ -171,14 +210,22 @@ governing permissions and limitations under the License.
   &:after {
     content: '';
     position: absolute;
-    inset: calc(-1 * var(--spectrum-global-dimension-size-50));
+    inset: calc(-2 * var(--mod-swatch-focus-ring-gap, var(--spectrum-swatch-focus-ring-gap)));
 
     opacity: 0;
 
-    border: var(--spectrum-global-dimension-static-size-25) solid var(--spectrum-alias-focus-ring-color);
-    border-radius: var(--spectrum-alias-focus-ring-border-radius-regular);
+    border-width: var(--mod-swatch-focus-ring-thickness, var(--spectrum-swatch-focus-ring-thickness));
+    border-style: solid;
+    border-color: var(--highcontrast-swatch-focus-ring-color, var(--mod-swatch-focus-ring-color, var(--spectrum-swatch-focus-ring-color)));
+    border-radius: var(--mod-swatch-focus-ring-border-radius, var(--mod-swatch-focus-ring-border-radius, var(--spectrum-swatch-focus-ring-border-radius)));
```

Comment:
Looks like this has some redundancies, should be
`border-radius: var(--mod-swatch-focus-ring-border-radius, var(--spectrum-swatch-focus-ring-border-radius));`